rilldata/rill
# Fix pressing back button to default dashboard

## Checklist

## Summary
#### Issue addressed: 
closes #2536

#### Details:
When switching to the default state we were not synching the default url state to the client side state.

## Steps to Verify
- Go to any dashboard.
- Selecting a few filters in a dashboard.
- Click back button until all filters are empty.
- Back and forward should take you through the different filters.
